website/src/pages/documents/document.page.tsx

```diff
@@ -490,23 +481,28 @@ export const DocumentTitleHeader = (p: {
           ) : null}
         </div>
       </div>
-      {p.doc.audioRecording && ( // TODO Implement sticky audio bar
-        <div id="document-audio-player" className={css.audioContainer}>
-          <span>Document Audio:</span>
-          <AudioPlayer
-            style={{ flex: 1 }}
-            audioUrl={p.doc.audioRecording.resourceUrl}
-            showProgress
-          />
-          {p.doc.audioRecording && !isMobile && (
-            <div>
-              <a href={p.doc.audioRecording?.resourceUrl}>
-                <Button>Download Audio</Button>
-              </a>
-            </div>
-          )}
-        </div>
-      )}
+      {p.doc.editedAudio.length > 0 && // TODO Implement sticky audio bar
+        p.doc.editedAudio.map((audio, index) => (
+          <div
+            id="document-audio-player"
+            className={css.audioContainer}
+            key={index}
+          >
+            <span>Document Audio:</span>
+            <AudioPlayer
+              style={{ flex: 1 }}
+              audioUrl={audio.resourceUrl}
+              showProgress
+            />
+            {!isMobile && (
+              <div>
+                <a href={audio.resourceUrl}>
+                  <Button>Download Audio</Button>
+                </a>
+              </div>
+            )}
+          </div>
+        ))}
```

Comment:
For now, we are rendering the same document audio section over and over for each piece of edited audio.